### PR TITLE
Admin Actions Log: 2024 - February 10

### PR DESCRIPTION
**Pull request by confused-Techie**
### Requirements 

* Filling out the template is required.
* All new code requires tests to ensure against regressions.
  - However, if your PR contains zero code changes, feel free to select the checkmark below to indicate so.

* [ ] Have you ran tests against this code?
* [X] This PR contains zero code changes.

### Description of the Change

While checking our logs on the backend, I discovered that a package had malformed data, rendering it uninstallable.

This actions log details the issue, and it's resolution.

### CODE REVIEW

File: docs/reference/Admin_Actions.md [hunk 1]
```diff
@@ -4,6 +4,22 @@ When you consider that most backend services are a black box of code and decisio
 
 With that said this document will serve as the ongoing history of administrative actions that must be taken against the backend.
 
+## 2024 - February 10
+
+### auto-save-on-idle
+
+The community package [`auto-save-on-idle`](https://web.pulsar-edit.dev/packages/auto-save-on-idle) was discovered to have malformed data on the PPR database, rendering it's most recent version uninstallable. This was discovered via automatic error reporting on the backend.
```
Comment (Daeraxa) [inline docs/reference/Admin_Actions.md:11]:
```suggestion
The community package [`auto-save-on-idle`](https://web.pulsar-edit.dev/packages/auto-save-on-idle) was discovered to have malformed data on the PPR database, rendering its most recent version uninstallable. This was discovered via automatic error reporting on the backend.
```

File: docs/reference/Admin_Actions.md [hunk 2]
```diff
@@ -4,6 +4,22 @@ When you consider that most backend services are a black box of code and decisio
...
+
+This package was one of the original package's from Atom that was migrated over to the new system, and it seems that their last publication in 2017 was published under the GitHub tag of `v1.2.2` meanwhile, the `package.json` still had the incorrect version of `1.2.1`.
```
Comment (Daeraxa) [inline docs/reference/Admin_Actions.md:13]:
```suggestion
This package was one of the original packages from Atom that was migrated over to the new system, and it seems that their last publication in 2017 was published under the GitHub tag of `v1.2.2` meanwhile, the `package.json` still had the incorrect version of `1.2.1`.
```

File: docs/reference/Admin_Actions.md [hunk 3]
```diff
@@ -4,6 +4,22 @@ When you consider that most backend services are a black box of code and decisio
...
+
+What this means is that during migration we attempted to find the data for this package's tag `v1.2.1` which doesn't exist, which meant that no data was ever saved for how to install this package. Which in term made it uninstallable.
```
Comment (Daeraxa) [inline docs/reference/Admin_Actions.md:15]:
```suggestion
What this means is that during migration we attempted to find the data for this package's tag `v1.2.1` which doesn't exist, which meant that no data was ever saved for how to install this package, which, in tern, made it uninstallable.
```

File: docs/reference/Admin_Actions.md [hunk 4]
```diff
@@ -4,6 +4,22 @@ When you consider that most backend services are a black box of code and decisio
...
+
+Considering this package has `42,347` downloads under it's belt, it seems safe to assume that this disconnect, something no longer allowed on the PPR, was something allowed on the original Atom backend.
```
Comment (Daeraxa) [inline docs/reference/Admin_Actions.md:17]:
```suggestion
Considering this package has `42,347` downloads under its belt, it seems safe to assume that this disconnect, something no longer allowed on the PPR, was something allowed on the original Atom backend.
```

File: docs/reference/Admin_Actions.md [hunk 5]
```diff
@@ -4,6 +4,22 @@ When you consider that most backend services are a black box of code and decisio
...
+
+In prioritizing keeping package's working, this package has had it's version `1.2.1` data updated, to include installable location data to it's GitHub tag of `v1.2.2`, as well as updating it's SHA information for that tag.
```
Comment (Daeraxa) [inline docs/reference/Admin_Actions.md:19]:
```suggestion
In prioritizing keeping packages working, this package has had its version `1.2.1` data updated to include installable location data to its GitHub tag of `v1.2.2`, as well as updating its SHA information for that tag.
```

**Review by Daeraxa — COMMENTED** [review]
Just a few corrections.

**Comment by confused-Techie** [comment]
Thanks for taking a look at this @Daeraxa!

**Pull request merged**